libr/util/idpool.c

```diff
@@ -163,6 +162,54 @@ R_API void* r_id_storage_get(RIDStorage* storage, ut32 id) {
 	return storage->data[id];
 }
 
+R_API bool r_id_storage_get_lowest(RIDStorage *storage, ut32 *id) {
+	r_return_val_if_fail (storage, false);
+	size_t i;
```

Comment:
why size_t, ids are ut32 and i is used as id here

Reply:
fixed